fsd_config/form_jsons/cof/eoi/cof-eoi-funding.json

```diff
@@ -292,131 +292,6 @@
-          "value": "Charitable incorporated organisation"
-        },
-        {
-          "text": "Co-operative (for example, a community benefit society)",
-          "value": "co-operative"
-        },
-        {
-          "text": "Community interest company (CIC)",
-          "value": "community-interest-company"
-        },
-        {
-          "text": "Company limited by guarantee",
-          "value": "company-limited"
-        },
-        {
-          "text": "Scottish charitable incorporate organisation (SCIO)",
-          "value": "scottish-charitable "
-        },
-        {
-          "text": "Parish, town and community councils",
-          "value": "parish-town-community-councils"
-        },
-        {
-          "text": "Trust port",
-          "value": "trust-port"
-        },
-        {
-          "text": "Not yet incorporated",
-          "value": "not-yet-incorporated"
-        }
-      ]
-    },
-    {
-      "title": "Type of asset",
-      "name": "dXSWRl",
-      "type": "string",
-      "items": [
-        {
-          "text": "Community centre",
-          "value": "Community centre"
-        },
-        {
-          "text": "Cinema",
-          "value": "Cinema"
-        },
-        {
-          "text": "Gallery",
-          "value": "Gallery"
-        },
-        {
-          "text": "Museum",
-          "value": "Museum"
-        },
-        {
-          "text": "Music venue",
-          "value": "Music venue"
-        },
-        {
-          "text": "Park",
-          "value": "Park"
-        },
-        {
-          "text": "Post office building",
-          "value": "Post office building"
-        },
-        {
-          "text": "Pub",
-          "value": "Pub"
-        },
-        {
-          "text": "Shop",
-          "value": "Shop"
-        },
-        {
-          "text": "Sporting or leisure facility",
-          "value": "Sporting or leisure facility"
-        },
-        {
-          "text": "Theatre",
-          "value": "Theatre"
-        },
-        {
-          "text": "Other",
-          "value": "Other"
-        }
-      ]
-    },
-    {
-      "title": "Risk to the asset",
-      "name": "ahxOlj",
-      "type": "string",
-      "items": [
-        {
-          "text": "Closure or end of lease",
-          "value": "Closure or end of lease"
-        },
-        {
-          "text": "Neglect or dereliction",
-          "value": "Neglect or dereliction"
-        },
-        {
-          "text": "Unsustainable current business model",
-          "value": "Unsustainable current business model"
-        },
-        {
-          "text": "Sale",
-          "value": "Sale"
-        },
-        {
-          "text": "Listed for disposal",
-          "value": "Listed for disposal"
-        },
-        {
-          "text": "Part of a community asset transfer",
-          "value": "Part of a community asset transfer"
-        }
-      ]
-    },
```

Comment:
`"title": "Stage purchasing asset",` (lines 420-442) can we remove also this part as it's not used in this form?

Reply:
Good spot 👍